### PR TITLE
Improving fault output documentation

### PR DESCRIPTION
**Pull request by AliceGabriel**

### CODE REVIEW

**Comment by fabian-kutschera** [comment]
LGTM (inspired by our discussion in the HFFZ manuscript)!
But I can't approve PR @Thomas-Ulrich.

File: Documentation/fault-output.rst [hunk 1]
```diff
@@ -99,35 +105,24 @@ The output is parametrized by the Pickpoint namelist, example:
...
-~~~~~~~~~~~
-
-SeisSol paraview files (XDMF/Hdf5 or XDMF/binary files, describing the fault outputs and the free-surface/volume wavefield) can also be read using our python module `seissolxdmf <https://pypi.org/project/seissolxdmf/>`__.
+This is the same as for the ParaView output.
 
 Additional Ascii output
 -----------------------
 
-The rupture front can be outputted at every gauss points by enabling RF_output_on.
+You can output the rupture front at every Gauss point by enabling **RF_output_on** in the DynamicRupture namelist:
```
Comment (Thomas-Ulrich) [inline Documentation/fault-output.rst:120]: This has been dropped out when porting DR to cpp.
Parameters currently read can be find in this file:
https://github.com/SeisSol/SeisSol/blob/master/src/DynamicRupture/Parameters.h

File: Documentation/fault-output.rst [hunk 2]
```diff
@@ -1,36 +1,41 @@
 .. _fault_output:
 
-Fault output
+Fault output - Dynamic Rupture Visualization Options
 ============
```
Comment (Thomas-Ulrich) [inline Documentation/fault-output.rst:4]: You need to extend the underline to fix the warnings:
```
(base) ulrich@ulrich-ThinkPad-T490s:~/SeisSol/SeisSol/Documentation$ make html
Running Sphinx v6.1.3
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
writing output... 
building [html]: targets for 5 source files that are out of date
updating environment: 0 added, 6 changed, 0 removed
reading sources... [100%] slip-rate-on-DR                                                                                                                                                                       
/home/ulrich/SeisSol/SeisSol/Documentation/fault-output.rst:4: WARNING: Title underline too short.

Fault output - Dynamic Rupture Visualization Options
============
/home/ulrich/SeisSol/SeisSol/Documentation/fault-output.rst:16: WARNING: Title underline too short.

DynamicRupture Namelist OutputPointType Configuration
------------
/home/ulrich/SeisSol/SeisSol/Documentation/fault-output.rst:16: WARNING: Title underline too short.

DynamicRupture Namelist OutputPointType Configuration
------------
/home/ulrich/SeisSol/SeisSol/Documentation/fault-output.rst:36: WARNING: Title underline too short.

Configuring ParaView output
---------------
/home/ulrich/SeisSol/SeisSol/Documentation/fault-output.rst:36: WARNING: Title underline too short.

Configuring ParaView output
---------------
/home/ulrich/SeisSol/SeisSol/Documentation/fault-output.rst:88: WARNING: Title underline too short.

SeisSolXdmf Python Module
---------------------
/home/ulrich/SeisSol/SeisSol/Documentation/fault-output.rst:88: WARNING: Title underline too short.

SeisSolXdmf Python Module
---------------------
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] slip-rate-on-DR                                                                                                                                                                        
generating indices... genindex done
writing additional pages... search done
copying images... [100%] LatexFigures/Yoffe_example.png                                                                                                                                                         
copying static files... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 7 warnings.

The HTML pages are in _build/html.
```

File: Documentation/fault-output.rst [hunk 3]
```diff
@@ -45,50 +50,51 @@ This output is parametrized by the Elementwise namelist, example:
 printIntervalCriterion
 ~~~~~~~~~~~~~~~~~~~~~~
 
-If printIntervalCriterion = 1, the output is generated every N time steps,
-where N is set by printInterval. This option only works with Global time
-stepping. If printIntervalCriterion = 2, output is generated every
-printtimeinterval_sec.
+
+- When **printIntervalCriterion** = 1, the output is generated every N time steps, where N is defined by **printInterval**. Note that this option is only compatible with Global time stepping. 
+- When **printIntervalCriterion** = 2, output is generated every **printtimeinterval_sec**.
```
Comment (Thomas-Ulrich) [inline Documentation/fault-output.rst:55]: this printIntervalCriterion has been dropped when porting DR to cpp.
Here are the parameters currently read:
https://github.com/SeisSol/SeisSol/blob/master/src/DynamicRupture/Output/ParametersInitializer.hpp#L87
So you only need to document `printtimeinterval_sec`

File: Documentation/fault-output.rst [hunk 4]
```diff
@@ -1,36 +1,41 @@
...
-Two types of outputs are available for imaging the rupture. The rupture
-characteristics can be assessed at a set of locations, using ASCII
-receiver files, or overall the whole fault using files that can be
-opened in ParaView. Threads or nodes can be dedicated to write this
-latter output (see :ref:`asynchronous-output`),
-but it is usually not necessary. The type of output generated depends on the value
-of the variable `OutputPointType` of the DynamicRupture namelist:
+There are two primary methods for visualizing on-fault rupture dynamics:
+1. Evaluation of rupture characteristics at specific on-fault locations via ASCII receiver files
+2. Visualization across the entire fault surface with files that can be opened with ParaView
```
Comment (Thomas-Ulrich) [inline Documentation/fault-output.rst:11]: This is how it renders
image

(you can open the file _build/html/fault-output.html after running `make html` in the Documentation folder).
You need to add an empty line before the line starting with 1.

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]
Thank you. 
I  suggest a few changes, e.g. concerning deprecated parameters.

File: Documentation/fault-output.rst (hunk 1 above)
Comment (AliceGabriel) [reply Documentation/fault-output.rst:120]: Sounds good!

**Comment by fabian-kutschera** [comment]
Hi @Thomas-Ulrich,
I was trying to push changes I made based on your comments to @AliceGabriel original PR.

However, it seems I don't have the correct access rights:

> ERROR: Permission to SeisSol/SeisSol.git denied to fabian-kutschera.
> fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights
> and the repository exists.

I used the command `git push origin HEAD:AliceGabriel-patch-1` as suggested by GitHub documentation.
Any ideas what I might have done wrong?

**Comment by Thomas-Ulrich** [comment]
Hi @fabian-kutschera,
I've updated your rights.

**Review by Thomas-Ulrich — APPROVED** [review]
Note that I've precise a few lines.
Have a look if you want before merging.

**Pull request merged**